lib/slack-ruby-bot/commands/base.rb

```diff
@@ -122,6 +131,36 @@ def finalize_routes!
           command command_name_from_class
         end
 
+        def match_attachments(data, route)
+          fields_to_scan = %i[pretext text title]
```

Comment:
Maybe this should be the third optional `fields` parameter here defaulted to these. Just thinking about the future as we pass this in dynamically from the `attachments` options.

Reply:
Will think about this.